package.json

```diff
@@ -23,16 +23,18 @@
     "@nextui-org/react": "^1.0.0-beta.12",
     "next": "^13.1.3-canary.2",
     "next-themes": "^0.2.1",
+    "nextra": "^2.2.13",
+    "nextra-theme-docs": "^2.2.13",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.4.0",
-    "react-markdown": "^8.0.3",
+    "react-markdown": "^8.0.5",
+    "react-syntax-highlighter": "^15.5.0",
     "remark-breaks": "^3.0.2",
     "remark-gfm": "^3.0.1",
     "swr": "^1.3.0"
   },
   "devDependencies": {
-    "purescript": "0.15.6",
```

Comment:
Having the compiler in there seems to be what people recommend and works for CI / vercel deployment.

https://discord.com/channels/864614189094928394/873274004704747600/1068528747084521613